Formula/apache-arrow-glib.rb

```diff
@@ -0,0 +1,34 @@
+class ApacheArrowGlib < Formula
+  desc "GObject Introspection files of Apache Arrow"
+  homepage "https://arrow.apache.org/"
+  head "https://github.com/apache/arrow.git"
+
+  stable do
+    url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.7.0/apache-arrow-0.7.0.tar.gz"
+    sha256 "5d8e976e0c8a5f39087e3cd4c39efd45ad8960ffc5207ed02e999d36dc5b0e79"
+  end
+
+  depends_on "apache-arrow"
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gobject-introspection"
+
+  def install
+    cd "c_glib" do
+      args = %W[
+        --prefix=#{prefix}
+        CC=#{ENV.cc}
+      ]
+      system "./configure", *args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    ENV['GEM_HOME'] = testpath/'gems'
+    system "gem", "install", "gobject-introspection"
+    system "ruby", "-rgi", "-e", "GI.load('Arrow')"
+  end
```

Comment:
How about adding C code test?

```ruby
  test do
    (testpath/"test.c").write <<-EOS.undent
      #include "arrow-glib/arrow-glib.h"
      int main(void)
      {
        GArrowInt32ArrayBuilder *builder;
        return 0;
      }
    EOS
    glib = Formula["glib"]
    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
    flags += %W[
      -I#{include}
      -I#{glib.opt_include}/glib-2.0
      -I#{glib.opt_lib}/glib-2.0/include
      -L#{lib}
      -lglib-2.0
      -larrow-glib
    ]
    system ENV.cc, "test.c", "-o", "test", *flags
    system "./test"
  end
```

Reply:
Can't we examine gobject-introspection things with C code test?

Reply:
For functionality test, we should use gi-based apache arrow glib binding in testing.
But for homebrew formula test, we should verify just installation about headers and libraries.